doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -815,7 +815,9 @@ The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/mod
 
 The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
 
-* |no_changes_yet_note|
+* Fixed:
+
+  * An issue where a validation of the correct, small image fails due to incorrect pointer arithmetic inside :c:func:`bl_validate_firmware`.
```

Comment:
This doesn't really belong under MCUboot, perhaps under `Other libraries`.

The error can also cause an incorrect image to validate correctly.

```suggestion
  * An issue in :c:func:`bl_validate_firmware` where the reset vector validation check was not working properly.
```

Reply:
It should probably be under a point like `Bootloader Libraries` or something.

@greg-fer Can maybe give you the proper heading name.

Reply:
Would adding the following section before the Modem libraries section work?

```
Bluetooth libraries and services
--------------------------------

* :ref:`doc_bl_validation`:

  *  Fixed an issue in :c:func:`bl_validate_firmware` where the reset vector validation check would not work properly.
```

Reply:
Works for me. Change applied. @oyvindronningstad What about you?

Reply:
LGTM!